Packs/ContentTesting/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,57 @@
+
+#### Incident Types
+
+- **UnitTesting**
+
+#### Layouts
+
+##### UnitTestLayout
+
+- %%UPDATE_RN%%
+
+#### Scripts
+
+##### UpgradeCheck
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+##### UnitTest
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+##### UnitTestPBATaskMax
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+##### UnitTestLoadFields
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+##### New: ContentDependencies
+- Provides playbook to sub-playbook, playbook to automation, automation to automation dependencies
+- Can select Markdown and incident field for results output
+- Can select CSV and file name for results output
+- Searches a set of playbooks, or all playbooks if no playbook query is provided, and identifies all dependencies between playbooks and automations and outputs either CSV or markdown.  WARNING: Should be run on DEV since it may consume 100% CPU for 10 or more minutes if all playbooks are searched.
+
+This automation should be run in dev environments. It will consume 100% CPU for 10 minutes or more - depending on the amount of playbook and automation content.  It is set to timeout at 15 minutes in Advanced settings. This may need to be increased if content amount is large.  It only parses python scripts to see if additional automations/commands are invoked via demisto.executeCommand or execute_command. Javascript automations are not parsed.  Command names passed in a variable to demisto.executeCommand or execute_command are not reported.  If a python automation fails to parse, an error is reported in the war room: any automations it calls are not reported.
+Integration commands and builtins are not parsed (Available from Cortex XSOAR 6.8.0).
+##### UnitTestPlaybookAnalyzer
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+##### UnitTestCoverage
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+##### ChangeHistory
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+##### UnitTestPBAStats
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
+##### UnitTestPBATaskAvg
+
+- %%UPDATE_RN%%
+- Updated the Docker image to: *demisto/python3:3.10.12.62631*.
```

Comment:
Currently there is a bug in the release notes. Here are the release notes the contributor provided:

```
      "release_notes": "#### Incident Types\n##### UnitTesting\n- Updated to use new layout reference\n\n#### Layouts\n##### UnitTestLayout\n- Updated section name to 'Analysis Summary’\n- Updated button arguments to support new features in automations\n\n#### Scripts\n##### ContentDependencies\n- Provides playbook to sub-playbook, playbook to automation, automation to automation dependencies\n- Can select Markdown and incident field for results output\n- Can select CSV and file name for results output\n\n##### UpgradeCheck\n- Added support to filter on list of Content Packs\n\n##### UnitTestPlaybookAnalyzer\n- Added support for sub-playbook selection for the analysis\n- Added support for time range specification\n- Added support for maximum count of incidents\n- Simplified summary output since dependencies now handled in ContentDependencies\n\n",
```